docs/update-design-question.md

```diff
@@ -0,0 +1,45 @@
+Guys, we have a question for Formica where everyone is invited to participate deciding.
+
+So, formica currently is a tool for managing groups of units. Example:
+
+```
+$ formicactl submit elasticsearch@{1,2}
+$ formciactl start elasticsearch@{1,2}
+$ formicactl status elasticsearch
+Group           Unit  <more headers>
+elasticsearch@1 *     active  active  running 10.0.4.100
+elasticsearch@2 *     active  active  running 10.0.4.101
+```
+
+As you see, it is very similar to `fleetctl`. Next we want to implement an update
+functionality, similar (but better ;) than what `releaseit` currently does.
+
+## Question 1
+In the example above we have used `1` and `2` for the slices. When updating, we
+want the users to have the option to create new instances first. This raises where
+the names for those slices come from.
+
+* Option 1: Just increment the found ids. So if we find `7,8,9` we continue with
+  `10, 11, ..`
+* Option 2: Use random IDs. This would imply changes to the idea that user dictates
+  these names as shown above.
+* Option 3: Extend the slice with a timestamp so we can reuse the old slice but still differentiate
+  them from existing slices.
+
+## Question 2
+
+The second question is how the update command it self looks like.
+
+1. `$ formicactl update --sleep-time=1m --max-unavailable=50% --max-surge=100% elasticsearch`
```

Comment:
sleep time in our case is to give our ambassadors a time actually work. Kubernetes has sthg similar, where they e.g. wait for 1min to and verify the new container actually stayed up in that time (and didn't die again).